src/App.css

```diff
@@ -0,0 +1,285 @@
+  height: 4.25rem;
+  color: var(--medium-blue);
+}
+
+.book-progress {
+  display: flex;
+  align-items: center;
+}
+
+.text-peogress {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.Percent-Complete {
+  margin: 0.063rem 5.75rem 0 0.688rem;
+  font-family: Montserrat, Arial, Helvetica, sans-serif;
+  font-size: 2rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.Complete-text {
+  margin: 0.438rem 4.938rem 0.75rem 0.75rem;
+  opacity: 0.5;
+  font-family: Montserrat, Arial, Helvetica, sans-serif;
+  font-size: 0.875rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.chapter-prpgress {
+  display: flex;
+  flex-direction: column;
+}
+
+.update-progress {
+  margin: 0.75rem 1.75rem 0.563rem 3.688rem;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border: none;
+  border-radius: 3px;
+  background-color: var(--azure);
+  font-family: RobotoSlab, Arial, Helvetica, sans-serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: var(--white-two);
+}
+
+.Current-Chapter,
+.Current-Lesson {
+  margin: 0.063rem 3.625rem 0 3.688rem;
+  opacity: 0.5;
+  font-family: RobotoSlab, Arial, Helvetica, sans-serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.Current-Lesson {
+  margin: 0.438rem 6.813rem 0.25rem 3.688rem;
+  font-size: 1rem;
+  color: #121212;
+  opacity: 1;
+}
+
+.form-container {
+  margin: 2.313rem 6.25rem 0;
+  border-top: 1px solid var(--white);
+}
+
+.form-container h2 {
+  margin: 1.813rem 32.75rem 1.188rem 0;
+  font-family: Montserrat, Arial, Helvetica, sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.inp-title,
+.inp-author {
+  margin: 1.188rem 5% 0 0;
+  padding: 0.813rem 35% 0.813rem 1.063rem;
+  border-radius: 4px;
+  border: solid 1px var(--white);
+  background-color: var(--white-two);
+  font-family: Montserrat, Arial, Helvetica, sans-serif;
+  font-size: 1rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.15px;
+}
+
+.inp-author {
+  padding: 0.813rem 0.938rem 0.813rem 1rem;
+}
+
+.add-btn {
+  width: 12%;
+  height: 2.813rem;
+  border: none;
+  border-radius: 3px;
+  background-color: var(--azure);
+  font-family: RobotoSlab, Arial, Helvetica, sans-serif;
+  font-size: 0.813rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: var(--white-two);
+}
```

Comment:
- **[OPTIONAL]** Well done implementing the required changes, but it doesn't look like the original design yet. Please 🙏🏽 align the add button with the inputs on the form.

![image](https://user-images.githubusercontent.com/58642949/230721073-2f8777ec-7078-4514-a1be-70dd72d7ee77.png)